vpr/src/server/commconstants.h

```diff
@@ -0,0 +1,38 @@
+#ifndef COMMCONSTS_H
+#define COMMCONSTS_H
+
+namespace comm {
+
+constexpr const char* KEY_JOB_ID = "JOB_ID";
+constexpr const char* KEY_CMD = "CMD";
+constexpr const char* KEY_OPTIONS = "OPTIONS";
+constexpr const char* KEY_DATA = "DATA";
+constexpr const char* KEY_STATUS = "STATUS";
+constexpr const char* ECHO_DATA = "ECHO";
+
+const unsigned char ZLIB_COMPRESSOR_ID = 'z';
+const unsigned char NONE_COMPRESSOR_ID = '\x0';
+
+constexpr const char* OPTION_PATH_NUM = "path_num";
+constexpr const char* OPTION_PATH_TYPE = "path_type";
+constexpr const char* OPTION_DETAILS_LEVEL = "details_level";
+constexpr const char* OPTION_IS_FLAT_ROUTING = "is_flat_routing";
+constexpr const char* OPTION_PATH_ELEMENTS = "path_elements";
+constexpr const char* OPTION_HIGHLIGHT_MODE = "high_light_mode";
+constexpr const char* OPTION_DRAW_PATH_CONTOUR = "draw_path_contour";
```

Comment:
resolved https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/2526/commits/7a84ac6c2f1adcf5b7dfa67c787d0ab81d2091c1

Reply:
@soheilshahrouz
i rework this part to use extern std::string instead of std::string_view.
https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/2526/commits/5b440ec54b44d27755da29bc7853b5f4d9de0854